pyflagser/__init__.py

```diff
@@ -1,17 +1,26 @@
 from ._version import __version__
 
-from .flagio import load_unweighted_flag, load_weighted_flag, \
-    save_unweighted_flag, save_weighted_flag
-from .flagser import flagser_unweighted, flagser_weighted
-from .flagser_count import flagser_count_unweighted, \
-    flagser_count_weighted
+__all__ = ['__version__']
 
-__all__ = ['load_unweighted_flag',
-           'load_weighted_flag',
-           'save_unweighted_flag',
-           'save_weighted_flag',
-           'flagser_unweighted',
-           'flagser_weighted',
-           'flagser_count_unweighted',
-           'flagser_count_weighted',
-           '__version__']
+# Allows to have access to the version information without
+# installating package + dependencies
+try:
+    from .flagio import load_unweighted_flag, load_weighted_flag, \
+        save_unweighted_flag, save_weighted_flag
+    from .flagser import flagser_unweighted, flagser_weighted
+    from .flagser_count import flagser_count_unweighted, \
+        flagser_count_weighted
+
+    __all__ += ['load_unweighted_flag',
+                'load_weighted_flag',
+                'save_unweighted_flag',
+                'save_weighted_flag',
+                'flagser_unweighted',
+                'flagser_weighted',
+                'flagser_count_unweighted',
+                'flagser_count_weighted',
+                'modules',
+                'tests'
```

Comment:
Can I ask you to make a build with these two lines removed, an the absolute import in `conftest` back as a relative import?  The reason is that this worked with Azure, and I don't see why it shouldn't work with GitHub actions.

Reply:
Done,
If CI and Wheels don't rise any other issues, I let you do a last review and I think we should be able to merge, what do you think @ulupo ?